tests/fail/unaligned_pointers/alignment.rs

```diff
@@ -8,7 +7,7 @@ fn main() {
     let x_ptr: *mut u8 = x.as_mut_ptr();
     // At least one of these is definitely unaligned.
     unsafe {
-        *(x_ptr as *mut u32) = 42;
-        *(x_ptr.add(1) as *mut u32) = 42;
+        *(x_ptr as *mut u32) = 42; //<0>~ ERROR: but alignment 4 is required
+        *(x_ptr.add(1) as *mut u32) = 42; //<0>~
```

Comment:
FWIW, I would have just put these two on the same line and added `rustfmt::skip` to it, which would need no code changes in ui_test... but now that you already did all that work I guess we can keep it. ;)

Reply:
Hmm... but we have to maintain it... so yea, just putting them on the same line it is